dev/cnf/oss_dependencies.maven

```diff
@@ -153,6 +153,7 @@ jakarta.transaction:jakarta.transaction-api:2.0.0-RC1
 jakarta.xml.bind:jakarta.xml.bind-api:3.0.0-RC3
 jakarta.xml.soap:jakarta.xml.soap-api:2.0.0-RC3
 jakarta.xml.ws:jakarta.xml.ws-api:3.0.0-RC3
+jakarta.validation:jakarta.validation-api:3.0.0-M1
```

Comment:
I really am going to write some gradle that fails the build if this file isn't kept alphabetical :-o